mod/wet4/views/default/page/layouts/elements/header.php

```diff
@@ -64,6 +74,6 @@
             }
         }
         $format_title = elgg_view_title($vars['title'], array('class' => 'elgg-heading-main mrgn-lft-sm'));
-        echo elgg_format_element('div', ['class' => 'd-flex title-button-combo'], $format_title .'<div class="title-action-button d-flex">' . $buttons2 . $buttons . '</div>');
+        echo elgg_format_element('div', ['class' => 'd-flex title-button-combo'], $format_title .'<div class="title-action-button d-flex">' . $buttons2 . $buttons . $notificationSettingsBtn . '</div>');
```

Comment:
I think the position of the settings button is good. Maybe we can add a `btn-default` style class it make it look like a secondary button. This way the main action visually still looks like compose.

Reply:
I think that looks a lot better, the only question I have now is: do we want to keep the settings link on all of the messages pages (inbox, notifications, and messages received)? Personally I think it's fine since it makes about as much sense as having compose message on all of them too and I don't think it gets in the way really.

Reply:
Yes I think it's fine as is. I think having the link to the settings available on the page is overall a positive UX fix to direct users to their settings. I'll approve this PR.